README.md

```diff
@@ -2143,6 +2143,72 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='opaque-generic-parameters'></a>(<a href='#opaque-generic-parameters'>link</a>) **Prefer using opaque generic parameters (with `some`) over verbose named generic parameter syntax where possible.**  [![SwiftFormat: opaqueGenericParameters](https://img.shields.io/badge/SwiftFormat-opaqueGenericParameters-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#opaqueGenericParameters)
```

Comment:
Link to proposal in case it's helpful: https://github.com/apple/swift-evolution/blob/main/proposals/0341-opaque-parameters.md